apps/emqx_bridge_rocketmq/src/emqx_bridge_rocketmq.erl

```diff
@@ -58,15 +61,144 @@ values(post) ->
             max_buffer_bytes => ?DEFAULT_BUFFER_BYTES
         }
     };
-values(put) ->
-    values(post).
+conn_bridge_example_values(put) ->
+    conn_bridge_example_values(post).
+
+%% TODO fix these examples
```

Comment:
Is this TODO still applicable?

Reply:
Fixed: [976099f](https://github.com/emqx/emqx/pull/12488/commits/976099f5fbb3f1894b77d9017465bef114ff4eef)